pymc/distributions/continuous.py

```diff
@@ -848,10 +849,12 @@ class HalfNormal(PositiveContinuous):
 
     Parameters
     ----------
-    sigma: float
+    sigma : tensor_like of float, optional
         Scale parameter :math:`sigma` (``sigma`` > 0) (only required if ``tau`` is not specified).
```

Comment:
```suggestion
        Scale parameter :math:`\sigma` (``sigma`` > 0) (only required if ``tau`` is not specified).
```

this wasn't rendering correctly, it should match the notation used in the extended description